include/ublk_cmd.h

```diff
@@ -28,19 +28,22 @@
  *      this IO request, request's handling result is committed to ublk
  *      driver, meantime FETCH_REQ is piggyback, and FETCH_REQ has to be
  *      handled before completing io request.
+ * 
+ * NEED_GET_DATA: only used for write requests to set io addr and copy data
+ *      without help of task_work.
+ * 		When NEED_GET_DATA is set, ublksrv has to issue UBLK_IO_NEED_GET_DATA
+ * 		command after ublk driver returns UBLK_IO_RES_NEED_GET_DATA.
  *
- * REFETCH_REQ: issued via sqe(URING_CMD) after ublk driver returns
- * 	UBLK_IO_REFETCH_REQ, which only purpose is to fetch io request data
- * 	from the userspace ublk server context in case that task_work_add
- * 	isn't available because ublk driver is built as module
+ *     	It is only used if ublkserver set UBLK_F_NEED_GET_DATA flag
+ * 		while starting a ublk device.
  */
 #define	UBLK_IO_FETCH_REQ		0x20
 #define	UBLK_IO_COMMIT_AND_FETCH_REQ	0x21
-#define	UBLK_IO_REFETCH_REQ		0x22
+#define UBLK_IO_NEED_GET_DATA	0x22
 
 /* only ABORT means that no re-fetch */
 #define UBLK_IO_RES_OK			0
-#define UBLK_IO_RES_REFETCH		1
+#define UBLK_IO_RES_NEED_GET_DATA	1
```

Comment:
Please pull the latest master branch in which UBLK_IO_RES_REFETCH has been removed days ago

Reply:
OK, I will use latest master branch.

Reply:
Yes, but UBLK_IO_REFETCH_REQ is removed.

Reply:
In https://github.com/ming1/ubdsrv/blob/master/include/ublk_cmd.h: 
#define	UBLK_IO_REFETCH_REQ		0x22
[...]
#define UBLK_IO_RES_REFETCH		1